tests/test_meshtastic_utils_ble.py

```
@@ -13,18 +13,44 @@
+    connect_meshtastic,
+)
 from tests.conftest import cleanup_ble_future_state
+from tests.constants import TEST_BLE_MAC
 
 TEST_PACKET_RX_TIME = 1234567890
 
 
+class _SuppressedBLEInterface:
+    """Test double that simulates duplicate connect suppression."""
+
+    def __init__(self, **_kwargs):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)